### DISCUSSION TITLE
Open post image in modal

### DISCUSSION THREAD

**Discussion by Varhal** [forem/forem#15370] [Feature Requests]
Hey! All modern platforms (medium, etc.) allow you to view the image in a modal window. This improves user experience, users will make fewer clicks to view large format images. Technically, it's easy to do.

I propose to make a maximally simple modal window for large format displays. Clicking on the image opens a modal window with an image, the second click closes it.

thanks

**Comment by Eduardo4646** [comment]
Hi, really appreciate your efforts that you developed code for the simple modal images. As I also heared about it that it make the user experience better. Do you have sample of it so that I could see it. Because I am going to hire someone to convert all the images of [fifa mobile mod apk](https://nyapk.com/fifa-mobile-mod-apk/) into the simple modal window images. If you can do it so let me know.

**Reply by Varhal** [reply]
Hi! No we didn't do it
